src/Ocelot.Provider.Polly/PollyQoSResiliencePipelineProvider.cs

```diff
@@ -48,11 +50,9 @@ private void PollyResiliencePipelineWrapperFactory(ResiliencePipelineBuilder<Htt
     {
         var options = route.QosOptions;
 
-        // Add TimeoutStrategy if TimeoutValue is not int.MaxValue and greater than 0
-        if (options.TimeoutValue != int.MaxValue && options.TimeoutValue > 0)
-        {
-            builder.AddTimeout(TimeSpan.FromMilliseconds(options.TimeoutValue));
-        }
+        builder.AddTimeout(options.TimeoutValue > 0
```

Comment:
not agree with that, if an user (like me) does not want to add a TimeOutStrategy, he can do it